tests/unit/update-manager-n3patch-insert-test.ts

```diff
@@ -97,13 +94,74 @@ _:patch
 
 _:patch
 
- 
       solid:inserts {
         _:subj <https://pod.example/test/foo#predicate> <https://pod.example/test/foo#subject> .
       };   a solid:InsertDeletePatch .
 `)
     });
 
+
+    it('Constructs n3patch document correctly', async () => {
+        // updater.anonymize = sinon.spy();
```

Comment:
what about the commented out lines (also some more of those below)? Can they go away or is this unfinished somehow?

Reply:
Now they are removed.